package.json

```diff
@@ -44,6 +45,7 @@
     "@lhci/cli": "^0.12.0",
     "beachball": "^2.31.0",
     "cross-env": "^7.0.3",
-    "patch-package": "^6.4.7"
+    "patch-package": "^6.4.7",
+    "playwright": "1.40.0"
```

Comment:
Any thoughts on a policy for keeping this updated given that we can't rely on Renovate to do so?

1. Manually update it whenever we notice it's drastically out of date? 
2. Some kind of ongoing tech debt issue that reminds us to do it?
3. Policy to update it manually each month when completing Renovate PRs?

And any docs to write to help make the upgrade easier? Is it basically just "search package.json for playwright and manually change the version, making sure to keep them all aligned"?

Reply:
How about I create a recurring reminder on my calendar (every 3 months?) to create a tech debt issue for updating Playwright?

I don't think docs are necessary. It's exactly what you said, and if someone does something different, it should be easy to notice in the PR.

Reply:
If you're up for owning it, that sounds good to me! If it becomes too annoying we can always fall back to option 1.

Reply:
I added it to the Manual Regression Validations list: https://github.com/ni/nimble/issues/1747